### ISSUE TITLE
'RegexpTagger' object has no attribute '_regexps'

### ISSUE THREAD

**Issue by guizonatto** [nltk/nltk#2551]
Is the issue https://github.com/nltk/nltk/issues/287 back in the 3.5 version?

I test the same code in 3.4.5 and it works fine.

**Comment by tomaarsen** [comment]
I'm unable to reproduce this issue. The attribute `_regexps` is created in the first line after the `super().__init__` in `RegexpTagger`, so I don't see how this issue originated. 
For the record, this worked for me:
```python

from nltk.tag import RegexpTagger

regexp_tagger = RegexpTagger(
    [(r'^-?[0-9]+(.[0-9]+)?$', 'CD'),   # cardinal numbers
    (r'(The|the|A|a|An|an)$', 'AT'),   # articles
    (r'.*able$', 'JJ'),                # adjectives
    (r'.*ness$', 'NN'),                # nouns formed from adjectives
    (r'.*ly$', 'RB'),                  # adverbs
    (r'.*s$', 'NNS'),                  # plural nouns
    (r'.*ing$', 'VBG'),                # gerunds
    (r'.*ed$', 'VBD'),                 # past tense verbs
    (r'.*', 'NN')                      # nouns (default)
])

out = regexp_tagger.tag(["quizzable", "happiness", "an"])
print(out)
```
Which (correctly) produces:
```python
[('quizzable', 'JJ'), ('happiness', 'NN'), ('an', 'AT')]
```

If you are still experiencing this issue, we'd welcome information on how to reproduce it. Until then, I'll close this issue, under the assumption that it has been fixed since version 3.5.

**Comment by yaalaa** [comment]
The issue is [alive](https://github.com/nltk/nltk/issues/287#issuecomment-1546281902)
